### PR TITLE
spelling, capitalization, clarification

### PR DESCRIPTION
**Pull request by tkphd**
This PR fixes some spelling & capitalization stuff, and clarifies a thing or two, in MAINTENANCE.

### CODE REVIEW

File: MAINTENANCE.md [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
 
 #### Carpentries-Incubator Project
 
-HPC-Intro is currently on [GitHub][hpc-intro].  You need a github
-account to contribute.  In addition, in order to get write access to
-the repo, you will need to become a member of the HPC Intro
-Maintainers team, which lives inside the "carpentries-incubator"
+HPC-Intro is currently on [GitHub][hpc-intro]. You need a GitHub
+account to contribute. In addition, in order to get write access to
+the reposiroty ("repo"), you will need to become a member of the HPC
```
Comment (psteinb) [inline MAINTENANCE.md:17]:
```suggestion
the repository ("repo"), you will need to become a member of the HPC
```

**Review by psteinb — CHANGES_REQUESTED** [review]

File: MAINTENANCE.md (hunk 1 above)
Comment (tkphd) [reply MAINTENANCE.md:17]: :rofl:

Comment (tkphd) [reply MAINTENANCE.md]: Fixt in 13b615e.

**Review by psteinb — APPROVED** [review]

**Comment by psteinb** [comment]
Thanks! As always ... :+1:

**Pull request merged**